.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ release_tag: &release_tag /^[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+)?$/
 defaults_js: &defaults_js
   <<: *defaults
   docker:
-    - image: cimg/node:12.22
+    - image: cimg/node:14.17.4
```

Comment:
Good catch!
I've checked it now and it appears that we don't really need a Node image since the scripts are using only Bash and Python 🤔 
Do you know why is using such an image?

Reply:
https://github.com/pagopa/io-app/pull/3402/commits/b70219edd011a9e265d08814570559855c16c5b8

Trying with `python:3.8` like the other python job.

Reply:
Yes, that was my fault, definitely better!